doc/sphinx/reference/vcl_var.rst

```diff
@@ -1054,9 +1054,9 @@ beresp.do_esi
 
 	Type: BOOL
 
-	Readable from: vcl_backend_response, vcl_backend_error
+	Readable from: vcl_backend_response, vcl_backend_error, vcl_backend_refresh
 
-	Writable from: vcl_backend_response, vcl_backend_error
+	Writable from: vcl_backend_response, vcl_backend_error, vcl_backend_refresh
```

Comment:
do we want to spread do_XXX? I would have thought that as `vcl_backend_refresh` is called in addition, we should only use to decide on the resulting beresp and keep the other logic to `vcl_backend_response`.

(comment applies to the following, too)

Reply:
bugwash could not come up with strong reasons for or against.
My worry is that this might be confusing, but it probably is not an important point.